plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -115,7 +115,7 @@ func Run(s *options.SchedulerServer) error {
 		glog.Fatal(server.ListenAndServe())
 	}()
 
-	configFactory := factory.NewConfigFactory(kubeClient, s.SchedulerName, s.HardPodAffinitySymmetricWeight, s.FailureDomains)
+	configFactory := factory.NewConfigFactory(leaderElectionClient, s.SchedulerName, s.HardPodAffinitySymmetricWeight, s.FailureDomains)
```

Comment:
Nope, unversioned client (1) differs from clientset core client (2). Reverting the change.

1) ``EventInterface.Patch(event *api.Event, data []byte) (*api.Event, error)``
2) ``EventInterface.Patch(name string, pt api.PatchType, data []byte, subresources ...string) (result *api.Event, err error)``